docs/main/06-kratix-cli/20-reference/11_kratix-update-dependencies.md

```diff
@@ -0,0 +1,28 @@
+# kratix update dependencies
+Commands to update promise dependencies
+
+## Description
+Commands to update promise dependencies
```

Comment:
Could we mention that `update dependencies` automatically sets namespace to default when namespace is not set for all resources? this could come as a surprise for people.